bindings/ios/MEGAPushNotificationSettings.h

```diff
@@ -0,0 +1,145 @@
+ *  1.3. Do Not Disturb for a period of time: it overrides the schedule, if any (no notification will be generated)
+ *
+ * 2. Chats: Mute for all chats notifications
+ *
+ * 3. Per chat:
+ *  2.1. Mute all notifications from the specified chat
+ *  2.2. Always notify for the specified chat
+ *  2.3. Do Not Disturb for a period of time for the specified chat
+ *
+ * @note Notification settings per chat override any global notification setting.
+ * @note The DND mode per chat is not compatible with the option to always notify and viceversa.
+ *
+ * 4. Contacts: new incoming contact request, outgoing contact request accepted...
+ * 5. Shared folders: new shared folder, access removed...
+ *
+ */
+
+@interface MEGAPushNotificationSettings : NSObject
+
+/**
+ * @brief Getter returns the timestamp (in seconds since the Epoch) until the DND mode is enabled and setter can be used to set the global DND mode for a period of time.
+ *
+ * This property returns valid value only if the value of property globalDNDEnabled is YES.
+ * No notifications will be generated until the specified timestamp.
+ *
+ * If notifications were globally disabled, this function will enable them
+ * back (but will not generate notification until the specified timestamp).
+ *
+ * If there's no DND mode established, this property returns -1.
+ * @note a DND value of 0 means the DND does not expire.
+ *
+ */
+@property (assign, nonatomic) int64_t globalDNDTimestamp;
+
+/**
+ * @brief Returns whether Do-Not-Disturb mode is enabled or not
+ * @return YES if enabled, NO otherwise
+ */
+@property (readonly, nonatomic, getter=isGlobalDNDEnabled) BOOL globalDNDEnabled;
+
+/**
+ * @brief Returns whether Do-Not-Disturb mode for a chat is enabled or not
+ *
+ * @param chatId handle of the node that identifies the chat room
+ * @return YES if enabled, NO otherwise
+ */
+- (BOOL)isChatDndEnabledForChatId:(int64_t)chatId;
+
+/**
+ * @brief Enable or disable notifications for a chat
+ *
+ * If notifications for this chat are disabled, the DND settings for this chat,
+ * if any, will be cleared.
+ *
+ * @note Settings per chat override any global notification setting.
+ *
+ * @param enabled YES to enable, NO to disable
+ * @param chatId handle of the node that identifies the chat room
+ */
+- (void)setChatEnabled:(BOOL)enabled forChatId:(int64_t)chatId;
+
+/**
+ * @brief Returns the timestamp until the Do-Not-Disturb mode for a chat
+ *
+ * This method returns a valid value only if [MEGAPushNotificationSettings isChatEnabled]
+ * returns NO and [MEGAPushNotificationSettings isChatDndEnabled] returns YES.
+ *
+ * If there's no DND mode established for the specified chat, this function returns -1.
+ * @note a DND value of 0 means the DND does not expire.
+ *
+ * @param chatId handle of the Node that identifies the chat room
+ * @return timestamp until DND mode is enabled (in seconds since the Epoch)
+ */
+- (int64_t)chatDndForChatId:(int64_t)chatId;
```

Comment:
The method should begin with the property name of the return value, here looks like the return value is a chat.

I think we can rename this method to `timestampDndForChatId` or `chatDndTimestampForChatId`, What do you think @lancy-mega ?